src/cli.rs

```diff
@@ -0,0 +1,11 @@
+use structopt::StructOpt;
+
+#[derive(Debug, PartialEq, Eq, Clone, StructOpt)]
+pub struct Args {
+    #[structopt(short, long, default_value = "17")]
```

Comment:
By the way, let's set default value to 19. It seems more visually appealing. What do you think?